### PR TITLE
Add documentation for `mobile: installCertificate` command

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Depends on https://github.com/appium/appium-xcuitest-driver/pull/652

### CODE REVIEW

**Review by imurchie — APPROVED** [review]

File: docs/en/writing-running-appium/ios/ios-xctest-install-certificate.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Automating Installation of Self-Signed Certificate on iOS
+
+Unfortunately, Apple does not provide any command line options, which can help to install self-signed certificate on the device or Simulator. However, there is [over-the-air](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/iPhoneOTAConfiguration/Introduction/Introduction.html) enrollment technology, which allows to deploy several types of entities, including such certificates, by simply downloading specially prepared configuration files with built-in web browser. After the configuration is downloaded it can be installed and trusted by going through several simple wizard steps.
```
Comment (jlipps) [inline docs/en/writing-running-appium/ios/ios-xctest-install-certificate.md]: no need for comma after `options` (unlike in German)

Comment (jlipps) [reply docs/en/writing-running-appium/ios/ios-xctest-install-certificate.md]: `which allows the deployment of`

Comment (jlipps) [reply docs/en/writing-running-appium/ios/ios-xctest-install-certificate.md]: `with the built-in web browser`

**Review by jlipps — APPROVED** [review]
couple language nits

File: docs/en/writing-running-appium/ios/ios-xctest-install-certificate.md (hunk 1 above)
Comment (mykola-mokhnach) [reply docs/en/writing-running-appium/ios/ios-xctest-install-certificate.md]: > no need for comma after options (unlike in German)

how do you know? o_O

Comment (mykola-mokhnach) [reply docs/en/writing-running-appium/ios/ios-xctest-install-certificate.md]: I'm actually thinking in Ukrainian where grammar rules are even more complicated than in German. But thanks for the explanation. The perfection requires a lot of practice: "Übung macht den Meister"

Comment (jlipps) [reply docs/en/writing-running-appium/ios/ios-xctest-install-certificate.md]: NP! Unfortunately I know nothing about Ukrainian, sounds complicated though :-)

**Pull request merged**